statler_api/tests/test_plays.py

```diff
@@ -122,57 +121,9 @@ def testCanRefreshPlaysFromDatabase(self):
         play.save()
 
         # This updates the title of all plays in the database. This may be a bad idea, but for now it works.
-        PlayDAO.objects.update(title="New Title")
+        Play.objects.update(title="New Title")
 
         play.refresh_from_db()
         self.assertEqual("New Title", play.title)
 
 
-class PlayTestCase(TestCase):
```

Comment:
These are all conversion tests, which aren't meaningful now that the models have been merged.